x-pack/plugins/maps/public/layers/blended_vector_layer.ts

```diff
@@ -260,33 +260,40 @@ export class BlendedVectorLayer extends VectorLayer implements IVectorLayer {
       prevDataRequest: this.getDataRequest(dataRequestId),
       nextMeta: searchFilters,
     });
-    if (canSkipFetch) {
```

Comment:
This PR highlighted an error with blended vector layers that `super._syncData` is skipped when new source data is not needed. Skipping `super._syncData` could prevent style meta changes from gettin picked up and re-synced.